distributed/protocol/tests/test_numpy.py

```diff
@@ -217,7 +217,6 @@ def test_itemsize(dt, size):
 
 
 def test_compress_numpy():
-    pytest.importorskip("lz4")
```

Comment:
I'm curious, why this change?  If we didn't have lz4, snappy, or zstandard installed (all of which are optional I think) then I'd expect this to fail.  

The only compressor we have by default,  I think, is zlib, and we don't compress with that by default.

Reply:
Actually, if you have snappy but not lz4 it will succeed.
zstandard does not install itself as a default compressor.
Amended the tests to reflect this.